src/components/Widgets/Widget.tsx

```diff
@@ -37,6 +37,8 @@ export function Widget({ starterVariant }: WidgetProps) {
   const wagmiConfig = useConfig();
   const { isMultisigSigner, getMultisigWidgetConfig } = useMultisig();
   const { multisigWidget, multisigSdkConfig } = getMultisigWidgetConfig();
+  const isGasVariant =
+    window?.location?.href?.includes(TabsMap.Refuel.variant) ?? false;
```

Comment:
Another approach could be: "activeTab" from store. Could give that a try but not sure which one works better.

Reply:
Way cleaner, thank you!